CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 * Add spacing to cookie banner confirmation message ([PR #1936](https://github.com/alphagov/govuk_publishing_components/pull/1936))
 * Fix Sass warning for extending a compound selector ([PR #1933](https://github.com/alphagov/govuk_publishing_components/pull/1933))
 * Add default aria label for navigation items (PR [#1946](https://github.com/alphagov/govuk_publishing_components/pull/1946))
+* Adds Reorder-able lists component ([PR #1905](https://github.com/alphagov/govuk_publishing_components/pull/1905))
```

Comment:
We refer to this as "reorderable list" component in the component documentation so this should match that naming.

Reply:
done